ofl/bevan/DESCRIPTION.en_us.html

```diff
@@ -1 +1,2 @@
-<p>Bevan is a reworking of a traditional slab serif display typeface created by Heinrich Jost in the 1930s. In Bevan, Jost's earlier letter forms have been digitised and then reshaped for use as a webfont, the counters have been opened up a little and the stems optimised for use as bold display font in modern web browsers.</p>
\ No newline at end of file
+<p>Bevan is a reworking of a traditional slab serif display typeface created by Heinrich Jost in the 1930s. In Bevan, Jost's earlier letter forms have been digitised and then reshaped for use as a webfont, the counters have been opened up a little and the stems optimised for use as bold display font in modern web browsers.</p>
```

Comment:
Can you add a short sentence about what the upgrade is about?

Reply:
Since the oblique was design suring the "summer of type" program, I think the description should credit the contributors (students and tutors). cc @davelab6